metabase/metabase
# Cache dashboard when navigating back to it via "Back button"

Epic https://github.com/metabase/metabase/issues/29500

Improve the "back to dashboard" flow by using the already loaded dashboard data to omit sending the BE request. Because the data stored in the store is normalized, we need to denormalize it before passing through. 

How to test:
- Open a dashboard with questions
- Click on a question title
- Without modifying the question, click the back button
- Your dashboard should appear without `/api/dashboard/:id` request.
